### PR TITLE
Audacity 3.6.0 Release Compatibility

### PR DESCRIPTION
**Pull request by RyanMetcalfeInt8**

### CODE REVIEW

File: mod-openvino/OVMusicGenerationLLM.cpp [hunk 1]
```diff
@@ -291,6 +294,8 @@ bool EffectOVMusicGenerationLLM::Process(EffectInstance&, EffectSettings& settin
    // If they have chosen a stereo model, then most likely their intention /
    // expecation is to generate a stereo track. So, in that case we replace
    // the mono track that was added with a new stereo one.
+   //TODO: FIXME!
```
[the remark this answers is not in the document]
Comment (RyanMetcalfeInt8) [reply mod-openvino/OVMusicGenerationLLM.cpp:297]: Dang -- something that I already fixed. I fix this 😆

File: mod-openvino/OVNoiseSuppression.cpp [hunk 2]
```diff
@@ -475,6 +475,8 @@ bool EffectOVNoiseSuppression::Process(EffectInstance&, EffectSettings&)
       return false;
    }
 
+   std::cout << "Going into WaveTrack Selection loop!" << std::endl;
```
[the remark this answers is not in the document]
Comment (RyanMetcalfeInt8) [reply mod-openvino/OVNoiseSuppression.cpp:478]: Yes, absolutely should  be removed -- nice catch.

**Comment by RyanMetcalfeInt8** [comment]
Ok, those things should be resolved now.

File: mod-openvino/OVMusicGenerationLLM.cpp [hunk 3]
```diff
@@ -211,6 +212,7 @@ bool EffectOVMusicGenerationLLM::DoEffect(
    return EffectBase::DoEffect(settings, finder, projectRate, list, factory, selectedRegion, flags, pAccess);
 }
 
+#if 0
```
Comment (gblong1) [inline mod-openvino/OVMusicGenerationLLM.cpp:215]: Just noticed this one. #if 0 code should be removed, unless there is a strong reason to keep it

**Review by gblong1 — CHANGES_REQUESTED** [review]
Sorry, one more change, please.

File: mod-openvino/OVMusicGenerationLLM.cpp (hunk 3 above)
Comment (RyanMetcalfeInt8) [reply mod-openvino/OVMusicGenerationLLM.cpp]: got it, okay, done.

**Review by gblong1 — APPROVED** [review]
LGTM

**Pull request merged**